contents/handbook/company/management.md

```diff
@@ -6,54 +6,30 @@ showTitle: true
+2. Making sure your direct reports are happy and productive
+3. Acting as the [hiring manager](https://posthog.com/handbook/people/hiring-process#the-role-of-the-hiring-manager) for new roles in your team
+4. Creating good plans for new person onboarding and small team offsites
+5. Raising team performance concerns with the relevant member of the exec team if you need help
 
 That's it.
 
 A manager at PostHog is _not_ responsible for:
-1. Setting compensation (we have a [compensation calculator](/handbook/people/compensation) and increases are done occasionally/proactively by the people team).
-1. Setting tasks for their direct reports
-1. "Approving," whether that's projects, expenses, days off or accounts (people should have admin access by default to most things)
-
-### What does setting context mean?
-
-At PostHog, we exclusively hire people that are the best in their field.
-That means managers won't need to spend time telling their direct reports what to do.
-
-However, for those people to make the best decisions, they need context. The things a manager can do to set context include:
-- creating a roadmap that the team can work towards
-- helping someone figure out who else to talk to within PostHog
-- enabling the team to measure their impact
-- improving the process in which a team works (things like standups, reviews etc)
-- organizing a team offsite or other meetup to work in person
-
-The shift here, and the biggest difference between PostHog and other places, is that in the end it is up to the individual to make the decisions.
-All you can do as a manager is set context. From there, you'll have to trust that we've made the right hiring decisions and that the individual is able to execute on that. If they can't, we have a [generous severance policy](/handbook/people/compensation#severance).
-
-Decisions aren't just about buying a piece of software or choosing a color for a button. It's also about what to work on, what to invest time in, or where to take entire parts of our product.
-
-Again, we've hired the best people and have high talent density, so we trust everyone to make these kinds of decisions.
-
-As a manager, it's tempting to see yourself as the sole owner of all the information, and give it out sparingly. People will come to you often with questions (because they don't have the context) and when they do you'll get more validation that holding all the context yourself makes you an Important Person.
-
-What managers should aim for at PostHog is to make themselves obsolete. Share as much context as possible, in written form and in a public channel. That way everyone will be able to do their best work.
+1. Deciding compensation - we have a [compensation calculator](/handbook/people/compensation) and the [process](/handbook/people/compensation#pay-reviews) is managed by James & Tim
+2. Setting tasks for your direct reports
+3. Providing a [career progression](/handbook/people/career-progression) plan for your team
```

Comment:
Doesn't this conflict with the later `Bi-annual 1-1s to talk about longer term career plans (make sure you put these in the calendar!)` ?
If not what's the difference? Is this something we don't do at all? or who does this?

Reply:
Maybe I can clarify that the bi-annual 1-1s are basically a series of questions you ask the team member to get them to think about their own growth and you agree what they should focus on together. But a manager doesn't go 'this is your plan for growth at PostHog, if you do X you will be a senior engineer etc..'

Reply:
btw I've never seen a manager give a plan like that to someone before (including my experience with several managers in previous companies). And "if you do X you will be a senior engineer etc" is definitely something everyone avoided, as it's not up to the manager only to decide. The big difference is that 
1. the career progression info as a table of expectations exists at a company level that managers can share and provide context on.
2. the bi-annual 1:1 conversations about perf / career actually happen (together with performance review cycle, which we do at off-sites as 360 feedback, but it's just a bit more involved at big tech)

Reply:
Your manager is responsible for having these periodic career checkins with you and prompting you to think about it, but yes the person is ultimately responsible for their own progression